### PR TITLE
chat message

### PR DESCRIPTION
**Pull request by d-waite**

### CODE REVIEW

**Comment by d-waite** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9** <a href="https://app.graphite.dev/github/pr/Waite-Development/Chat-App/9" target="_blank">Graphite</a> 
    * **PR #10** <a href="https://app.graphite.dev/github/pr/Waite-Development/Chat-App/10" target="_blank">Graphite</a>  👈
      * **PR #11** <a href="https://app.graphite.dev/github/pr/Waite-Development/Chat-App/11" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Waite-Development/Chat-App/10?utm_source=stack-comment).

File: apps/client/.turbo/turbo-dev.log [hunk 1]
```diff
@@ -1,162 +0,0 @@
-
```
Comment (twaite) [inline apps/client/.turbo/turbo-dev.log:1]: hmm, this file should be ignored,  I thought I added it to the gitignore, but maybe you used this before syncing with my code?

**Review by twaite — CHANGES_REQUESTED** [review]
Just need to remove the turbo file, the other changes look fine

**Review by twaite — APPROVED** [review]

**Comment by d-waite** [comment]
### Merge activity

* **Oct 11, 9:16 PM**: @d-waite started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Waite-Development/Chat-App/10).
* **Oct 11, 9:17 PM**: @d-waite merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Waite-Development/Chat-App/10).

**Pull request merged**